test/unit/md-renderer/index.js

```diff
@@ -143,14 +147,40 @@ describe('MarkdownRenderer', () => {
         });
 
         it('should convert relative image paths to Zendesk ID paths', () => {
-            const document = new Document('path');
+            const document = createDocument({path: 'path'});
             sandbox.stub(document, 'findByPath').returns(12345);
-            const renderer = new MarkdownRenderer(document);
+            const renderer = new MarkdownRenderer(document, createDummyConfig());
             const markdown = '![alt text](images/image.jpg "Title Text")';
             const html = renderer.render(markdown);
 
             expect(html)
                 .to.include('src="/hc/article_attachments/12345/image.jpg"');
         });
+
+        it('should highlight code blocks if the `highlight` config option is set to true', () => {
```

Comment:
No, there are also tests for highlighting in `highlight.js`. This test is basically just to make sure the config option works.